recipes/arm-variant/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+arm_variant_type:  # [aarch64]
+  - sbsa           # [aarch64]
```

Comment:
Is there any mechanism that we could use to prioritize getting sbsa instead of tegra?

Reply:
We could use features. It's kind of a blunt tool. So it would only prioritize one and there wouldn't be an order with any others

Agree this could make sense, but left it out initially. If we agree generally it should be added, we can do this

Reply:
Added `track_features` to handle this